src/WinGet.RestSource.IntegrationTest/Common/EndHttpMethod.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="EndHttpMethod.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.WinGet.RestSource.IntegrationTest.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Endpoint http method.
+    /// </summary>
+    public enum EndHttpMethod
```

Comment:
>EndHttpMethod [](http://example.com/codeflow?start=16&length=13)

Make sure we have Put and Del as well

Reply:
The class was not needed initially, code has been refactored to eliminate the need for this class. Removed.